build/build.js

```diff
@@ -3,19 +3,15 @@ const
   babel = require('gulp-babel'),
   del = require('del');
 
-gulp.task('cleanLib', () => del('lib/*'));
+gulp.task('clean', () => del('lib/*'));
 
-gulp.task('clean', gulp.series(['cleanLib']));
+gulp.task('copyTypings', () => gulp.src('src/**/*.d.ts').pipe(gulp.dest('lib')));
 
-gulp.task('copyTypings', gulp.series(['cleanLib']), () => gulp.src('src/**/*.d.ts')
```

Comment:
@danielsuter Had to make some changes in the gradle build files regarding gulp.series. Somehow the entire lib folder was not created anymore. Now it looks fine again.